Docs/user_guide/modules/volumes.md

```diff
@@ -74,6 +74,8 @@ Note: Consumer file formats, such as jpg, png, and tiff are not well suited for
   the user and provided to the software that imports the images. If the information is not entered correctly
   then the images may appear distorted and measurements on the images may provide incorrect results.
 - No standard way of indicating slice order: data may be easily get corrupted due to incorrectly ordered or missing frames.
+- Also consider the [Image Stacks](https://github.com/SlicerMorph/S_2020/blob/master/Day_1/ImageStacks/ImageStacks.md) provided by the [SlicerMorph](https://slicermorph.github.io/) extension to work with large sets of high resolution images.

```

Comment:
Can you move this up higher, before the step-by-step instructions? Users should see that they can either use imagestacks or follow the steps below.